Enable external contributors to run without any setup (#1125)
Expensify/App: README.md

    </h1>
</div>

#### Table of contents
* [Local Development](#local-development)
* [Running The Tests](#running-the-tests)
* [Debugging](#debugging)
* [Structure of the app](#structure-of-the-app)
* [Philosophy](#Philosophy)
* [Deploying](#deploying)

#### Additional Reading
* [Contributing to Expensify.cash](./CONTRIBUTING.md)
* [Expensify Code of Conduct](./CODE_OF_CONDUCT.md)
* [Contributor License Agreement](./CLA.md)

----

# Local development
These instructions should get you set up ready to work on Expensify.cash 🙌

## Getting Started 
1. Install `node` & `npm`: `brew install node`
2. Install `watchman`: `brew install watchman`
3. Install dependencies: `npm install`

You can use any IDE or code editing tool for developing on any platform. Use your favorite!

## Running the web app 🕸

> * To run the **development web app**, run the: `npm run web`

Reviewer: ```suggestion
* To run the **development web app**: `npm run web`
```

* Changes applied to Javascript will be applied automatically via WebPack as configured in `webpack.dev.js`

## Running the iOS app 📱

> * To install the iOS dependencies, run: `npm install && cd ios/ && pod install`

Reviewer: I read this line and thought it be a nice addition to have this so they end up back where they started for the next command. Optionally, we could add an npm script like `npm run iosinstall`
```suggestion
* To install the iOS dependencies, run: `npm install && cd ios/ && pod install && cd ..`
```

> * To install the iOS dependencies, run: `npm install && cd ios/ && pod install && cd ..`

Reviewer 2: NAB, `npm install` is step 3 of getting started so not sure we need it here but it's fine

* To run a on a **Development Simulator**: `npm run ios`
* Changes applied to Javascript will be applied automatically, any changes to native code will require a recompile

## Running the Android app 🤖
* To install the Android dependencies, run: `npm install`, then `gradle` will install all linked dependencies
* To run a on a **Development Emulator**: `npm run android`
* Changes applied to Javascript will be applied automatically, any changes to native code will require a recompile

## Running the MacOS desktop app 🖥
* To run the **Development app**, run: `npm run desktop`, this will start a new Electron process running on your MacOS desktop in the `dist/Mac` folder.

## Troubleshooting
1. If you are having issues with **_Getting Started_**, please reference [React Native's Documentation](https://reactnative.dev/docs/environment-setup)
2. If you are running into issues communicating with the API please verify your `.env` file is [set up correctly](#getting-started) for the platform you are trying to run.

> **Note:** Expensify engineers please refer to [these additional instructions](https://stackoverflow.com/c/expensify/questions/7699/7700) for testing against the local API.

Reviewer: ```suggestion
**Note:** Expensify engineers that will be testing with the API in your local dev environment please refer to [these additional instructions](https://stackoverflow.com/c/expensify/questions/7699/7700).
```

It made a little more sense to me to word it this way.
